Apps/Inc/Tasks.h

```diff
@@ -26,8 +26,7 @@
 
 /**
  * Priority Definitions
- */
-#define TASK_FAULT_STATE_PRIO               1
+ */ 
 #define TASK_INIT_PRIO                      2
```

Comment:
Okay, I'll remember that! In this particular case, though, I think it's good to leave it because Micrium documentation says our tasks should have priorities between 2 and (max prio - 3)